src/duckstation-qt/gamelistwidget.cpp

```diff
@@ -2181,29 +2172,27 @@ void GameListListView::updateFixedColumnWidths()
 
   // Played time is a little trickier, since some locales might have longer words for "hours" and "minutes".
   setFixedColumnWidth(fm, GameListModel::Column_TimePlayed,
-                      std::max(width_for(qApp->translate("GameList", "%n seconds", "", 59)),
-                               std::max(width_for(qApp->translate("GameList", "%n minutes", "", 59)),
-                                        width_for(qApp->translate("GameList", "%n hours", "", 1000)))));
+                      std::max({width_for(qApp->translate("GameList", "%n seconds", "", 59)),
+                                width_for(qApp->translate("GameList", "%n minutes", "", 59)),
+                                width_for(qApp->translate("GameList", "%n hours", "", 1000))}));
 
   // And this is a monstrosity.
   setFixedColumnWidth(
     fm, GameListModel::Column_LastPlayed,
-    std::max(
-      width_for(qApp->translate("GameList", "Today")),
-      std::max(width_for(qApp->translate("GameList", "Yesterday")),
-               std::max(width_for(qApp->translate("GameList", "Never")),
-                        width_for(QtHost::FormatNumber(Host::NumberFormatType::ShortDate,
-                                                       static_cast<s64>(QDateTime::currentSecsSinceEpoch())))))));
+    std::max({width_for(qApp->translate("GameList", "Today")), width_for(qApp->translate("GameList", "Yesterday")),
+              width_for(qApp->translate("GameList", "Never")),
+              width_for(QtHost::FormatNumber(Host::NumberFormatType::ShortDate,
+                                             static_cast<s64>(QDateTime::currentSecsSinceEpoch())))}));
 
   // Assume 8 is the widest digit.
   const int size_width = std::max(width_for(QStringLiteral("%1 MB").arg(8888)), width_for(tr("Unknown")));
   setFixedColumnWidth(fm, GameListModel::Column_FileSize, size_width);
   setFixedColumnWidth(fm, GameListModel::Column_DataSize, size_width);
 
-  setFixedColumnWidth(GameListModel::Column_Icon, m_model->getIconColumnWidth());
-  setFixedColumnWidth(GameListModel::Column_Region, 55);
-  setFixedColumnWidth(GameListModel::Column_Achievements, 100);
-  setFixedColumnWidth(GameListModel::Column_Compatibility, 100);
+  setFixedColumnWidth(fm, GameListModel::Column_Icon, m_model->getIconSizeWithPadding());
+  setFixedColumnWidth(fm, GameListModel::Column_Region, FLAG_PIXMAP_SIZE.width());
+  setFixedColumnWidth(fm, GameListModel::Column_Achievements, 100);
+  setFixedColumnWidth(fm, GameListModel::Column_Compatibility, COMPATIBILITY_PIXMAP_SIZE.width());
```

Comment:
Basically just use the same width estimation algorithm for all columns, taking translations into account, sort indicator, etc... Not sure if there was a reason to ignore the header width for these 4 columns or it was just an oversight.